js/components/developer/create-account-screen/index.js

```diff
@@ -18,6 +19,19 @@ import PostcodeInput from '../../common/post-code-input';
 import CheckBoxPUL from '../../common/checkbox-with-text';
 import GlobalStyle from '../../common/globalStyle';
 
+// TODO Replace these standard values with real values.
+// These are used during development only.
+function getCreateUserJson(email, password) {
+  return createJsonDataAttributes({
+    email,
+    password,
+    consent: true,
+    system_language_id: 38,
```

Comment:
_FYU_ The available system languages can be fetched from the API instead :) `api/v1/languages?filter[system_language]=true`